scripts/test_update.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+set -eo pipefail
+
+SCRIPTS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+CONTROLLER_BOOTSTRAP_DIR="$SCRIPTS_DIR/.."
+CONTROLLER_BOOTSTRAP="$CONTROLLER_BOOTSTRAP_DIR/bin/controller-bootstrap"
+NOTICE_TPL_FILE="$CONTROLLER_BOOTSTRAP_DIR/templates/NOTICE.tpl"
+AWS_SERVICE="eks"
+CONTROLLER_NAME="$AWS_SERVICE-controller"
+OUTPUT_DIR="$CONTROLLER_BOOTSTRAP_DIR/test_output"
+CONTROLLER_DIR="$OUTPUT_DIR/$CONTROLLER_NAME"
+
+if ! grep -wq -- "this line is added by ACK update test" "$CONTROLLER_DIR/NOTICE"; then
```

Comment:
I agree that "this line is added by ACK update test" is better than "xyz" :)

However, consider placing "this line is added by ACK update test" into a variable so that you don't need to repeat the string verbatim everywhere :)

```bash
TEXT_TO_FIND="this line is added by ACK update test"

if ! grep -wq -- "$TEXT_TO_FIND" "$CONTROLLER_DIR/NOTICE"; then
...
```

Reply:
Thank you for the suggestion :) this has been fixed.